kuma/landing/templates/landing/homepage.html

```diff
@@ -82,7 +82,7 @@ <h2 class="offscreen">{{ _('Web Technologies') }}</h2>
         </ul>
       </div>
       <div class="column-home-features">
-        <h3 class="zones"><i aria-hidden="true" class="icon-suitcase"></i><span>{{ _('Zones') }}</span></h3>
+        <h3 class="zones"><i aria-hidden="true" class="icon-suitcase"></i><span>{{ _('Mozilla<br> Docs') }}</span></h3>
```

Comment:
Should we use `{% trans %}Mozilla<br> Docs{% endtrans %}` here? (As we do with other i18n strings interspersed with markup.)